src/components/Modals/NewMessageModal.jsx

```diff
@@ -136,18 +143,26 @@ const NewMessageModal = ({ showModal, setShowModal, oldMessage = '', toField = '
             onSubmit={(e) => handleSubmit(e)}
             autoComplete="off"
           >
-            <TextField
-              margin="normal"
-              value={toField || message.recipientPodUrl}
-              type="text"
-              name="recipientPodUrl"
+            <Autocomplete
+              data-testid="newMessageTo"
+              freeSolo
+              value={recipientName?.person ?? message.recipientPodUrl}
+              disablePortal
+              autoSelect
               id="recipientPodUrl"
-              onChange={(e) => handleChange(e)}
+              options={contactListOptions}
+              onChange={(event, newValue) => {
+                setMessage({
+                  ...message,
+                  // if user wants to use a custom webId instead of a contact option, set the recipient value to the typed input
+                  recipientPodUrl: newValue.id ?? newValue
+                });
+              }}
+              fullWidth
               required
               autoFocus
-              label="To"
-              fullWidth
               disabled={toField !== ''}
+              renderInput={(params) => <TextField {...params} label="To*" />}
```

Comment:
Quick question, what's with the additional `*` for the `label`? Wouldn't `To` suffice?

Reply:
I'm actually not sure, what * is for. It shouldn't really be necessary. I'll mess around later to see and update it if it isn't.